Src/CSharpier/SyntaxPrinter/SeparatedSyntaxList.cs

```diff
@@ -11,8 +11,7 @@ public static class SeparatedSyntaxList
             SeparatedSyntaxList<T> list,
             Func<T, Doc> printFunc,
             Doc afterSeparator
-        )
-            where T : SyntaxNode {
+        ) where T : SyntaxNode {
```

Comment:
I remember seeing the change to pull the `where` clause into the same line, but isn't the `{` still supposed to be on a new line?

Reply:
Yeah it is, but the test cases all had empty bodies so it got missed
#299 will address it